helm/botkube/templates/deployment.yaml

```diff
@@ -109,18 +111,12 @@ spec:
               value: "{{.Release.Namespace}}"
             - name: BOTKUBE_SETTINGS_PERSISTENT__CONFIG_STARTUP_CONFIG__MAP_NAMESPACE
               value: "{{.Release.Namespace}}"
-            - name: BOTKUBE_SETTINGS_LIFECYCLE__SERVER_DEPLOYMENT_NAMESPACE
-              value: "{{.Release.Namespace}}"
-            - name: BOTKUBE_SETTINGS_LIFECYCLE__SERVER_DEPLOYMENT_NAME
-              value: "{{ include "botkube.fullname" . }}"
             - name: BOTKUBE_CONFIG__WATCHER_DEPLOYMENT_NAMESPACE
               value: "{{.Release.Namespace}}"
             - name: BOTKUBE_CONFIG__WATCHER_DEPLOYMENT_NAME
               value: "{{ include "botkube.fullname" . }}"
-            {{- if .Values.config.provider.endpoint }}
             - name: CONFIG_PROVIDER_ENDPOINT
               value: {{ .Values.config.provider.endpoint }}
-            {{- end }}
```

Comment:
Hey guys,
Nice job with this bot, really helpful 👍

I just saw the `{{- if .Values.config.provider.endpoint }}` was removed.
Any particular reason for it?
This is an old commit but this change is still on HEAD.

Thank you.

Reply:
Hi @joselcvarela  👋 

I'm glad you like Botkube! 🚀 

Regarding this change, while we could maintain it, we're currently using the default value here:https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml#L1228.

As long as you don't remove or override it, it should render properly. 
However, this endpoint will only be utilized if the `CONFIG_PROVIDER_IDENTIFIER` is set; otherwise, it will be ignored, and the local configuration will be used. In such cases, no external calls are executed. You can find the code responsible for this [here](https://github.com/kubeshop/botkube/blob/875f88c3c8d2cb06d0e6c09b3e16f3f9f3d3ef69/internal/config/remote/config.go#L24-L34).

Did you encounter any problems with it?

Reply:
Thank you for your response.
No, I haven't seen any issue so far.
I was just looking through the values passed to templates, and this part was intriguing me 👍
Keep on with the good job 🙌

Reply:
Thank you for your response.
No, I haven't seen any issue so far.
I was just looking through the values passed to templates, and this part was intriguing me 👍
Keep on with the good job 🙌